pelicanconf.py

```diff
@@ -21,6 +21,7 @@
     ('sprint', 'sprint'),
     ('meetup', 'meetup'),
     ('about', 'about'),    
+    ('project', 'project'),
```

Comment:
How about having `project` to be between `meetup` and `about`?

Reply:
agree